src/app/core/components/breadcrumbs/breadcrumbs.component.html

```diff
@@ -1,3 +1,3 @@
-<div class="container">
+<div class="container" *ngIf="isAuthenticated()">
```

Comment:
We have funny bug. If you feel all inputs and tap on LogIn button - we have breadcrumbs in "login" page. You are using routing, so you could just put "breadcrumbs" component into courses.